includes/activity/class-base-object.php

```diff
@@ -191,7 +191,7 @@ class Base_Object {
 	 *
 	 * @see https://www.w3.org/TR/activitystreams-vocabulary/#dfn-icon
 	 *
-	 * @var string
+	 * @var string|array
 	 *    | Image
```

Comment:
I think we have to either remove the following then, or put them also in line

Reply:
I can put them inline. Are you okay with removing ActivitiyPub spec types like Link or ObjectType in favor of PHP types?

Reply:
Sure!